src/script/common/c_content.cpp

```diff
@@ -1315,12 +1315,12 @@ void pushnode(lua_State *L, const MapNode &n)
 }
 
 /******************************************************************************/
-void warn_if_field_exists(lua_State *L, int table,
-		const char *name, const std::string &message)
+void warn_if_field_exists(lua_State *L, int table, const char *name, 
+		const std::string &nodename, const std::string &message)
```

Comment:
shouldn't be called "nodename" when it's also used for ore names

Reply:
Solved in https://github.com/minetest/minetest/pull/14794/commits/290e82562ade65e697aa16f3e599226eef9457f8